jwst/associations/lib/rules_level2b.py

```diff
@@ -618,7 +625,7 @@ def add_catalog_members(self):
                     logger.debug('Direct image %s has no EXPSPCIN defined.', direct)
                     continue
                 diff = direct_expspcin - expspcin
-                if diff > min_diff:
```

Comment:
This is the old logic that was resulting in the use of a matching direct image that's *furthest* in time from the grism image, instead of the one closest in time. So I've reversed the logic, while still having it select direct images that occur *after* the grism image.

Reply:
original author clearly thinks in reverse-space-time...

Reply:
Oooh, that's heavy